src/test/java/no/brreg/informasjonsforvaltning/organizationcatalogue/controller/OrganizationsApiImplTest.java

```diff
@@ -276,12 +280,27 @@ public void whenNonEmptyResult200() {
 
     @Nested
     class UpdateOrganization {
+        @Test
+        public void unauthorizedWhenNotAdmin() {
```

Comment:
Navnet på testen er litt misvisande når vi ser på kva for http-status som er forventa. (401 Unauthorized vs 403 Forbidden). Dersom eg ser på assert, ville eg ha gitt testen navnet forbiddenWhenNotAdmin.

Reply:
Ok, greit, kan endre navn. 

Men vi kan ikke teste unauthorized atm, må gå over til måten Rannveig setter opp på a-backend-service for å få til det. Jeg tenkte å vente til hun var ferdig før jeg gjorde det.